README.md

```diff
@@ -392,7 +392,7 @@ instance = SingleVMOpenEdXInstance.objects.create(
     sub_domain='dogwood.sandbox',
     name='Dogwood',
     fork_name='edx/edx-platform',
-    branch_name='named-release/dogwood',
+    branch_name='named-release/dogwood.rc',
```

Comment:
@antoviaque 

> I don't see an update to the README in #63 though - so it will be even more out of date after #63 is merged?

No, like I said

> the only thing that's left is for @bradenmacdonald to update the README

:)

cf. [my comment](https://github.com/open-craft/opencraft/pull/63/#issuecomment-219648634) on #63 about updating the README and [Braden's reply](https://github.com/open-craft/opencraft/pull/63/#issuecomment-219814653)